week_2/advanced_javascript/question_1/script.js

```diff
@@ -0,0 +1,15 @@
+const saveBtn = document.querySelector(".save__btn");
+
+
+saveBtn.addEventListener("click",() =>{
+    if(saveBtn.classList.contains("save__btn--red"))
+    {
+        saveBtn.classList.remove("save__btn--red");
+        saveBtn.classList.add("save__btn--black");
+        saveBtn.innerHTML = "Saved";
+    }else{
+        saveBtn.classList.remove("save__btn--black");
+        saveBtn.classList.add("save__btn--red");
+        saveBtn.innerHTML = "Save";
+    }
+});
```

Comment:
```suggestion
});

```
Good practice to add an empty line at the end of every file